exporter/exporterhelper/xexporterhelper/metadata.yaml

```diff
@@ -1,7 +1,7 @@
 type: xexporterhelper
 
 status:
-  class: exporter
```

Comment:
Means this is not generated

Reply:
I see what you mean. I should delete the generated files left behind.

Reply:
No, I got this wrong. You want these to generate test coverage? I am confused.

Reply:
I should stop working late nights. The comment was about confmap, because the value is invalid it means that there is no "go:generate" in the confmap directory.

Reply:
got it, I'll take a look.